src/SolutionTemplate/5.3/uno53AppWithLib/uno53AppWithLib/uno53AppWithLib.csproj

```diff
@@ -138,27 +138,27 @@
 
       <ItemGroup
           Condition="'$(TargetFramework)'=='net9.0-browserwasm'">
-        <_AssetsToValidate Include="$(PublishDir)wwwroot\Assets\SharedAssets.md" />
-        <_AssetsToValidate Include="$(PublishDir)wwwroot\Assets\json_file_asset.json" />
-        <_AssetsToValidate Include="$(PublishDir)wwwroot\Assets\Icons\icon_foreground.png" />
-        <_AssetsToValidate Include="$(PublishDir)wwwroot\Assets\Icons\icon_foreground.scale-100.png" />
-        <_AssetsToValidate Include="$(PublishDir)wwwroot\Assets\Icons\icon_foreground.scale-125.png" />
-        <_AssetsToValidate Include="$(PublishDir)wwwroot\Assets\Icons\icon_foreground.scale-150.png" />
-        <_AssetsToValidate Include="$(PublishDir)wwwroot\Assets\Icons\icon_foreground.scale-200.png" />
-        <_AssetsToValidate Include="$(PublishDir)wwwroot\Assets\Icons\icon_foreground.scale-300.png" />
-        <_AssetsToValidate Include="$(PublishDir)wwwroot\Assets\Icons\icon_foreground.scale-400.png" />
+        <_AssetsToValidate Include="$(PublishDir)wwwroot\$(WasmShellOutputPackagePath)/\Assets\SharedAssets.md" />
```

Comment:
is it normal that we have an extra `/` that isnt used else with the same variable?
```
Include="$(PublishDir)wwwroot\$(WasmShellOutputPackagePath)/\Assets\json_file_asset.json"
Include="$(PublishDir)wwwroot\$(WasmShellOutputPackagePath)\uno53lib\Assets\uno53lib-test.xml"
```

Reply:
nope :) Thanks!